citus-enterprise.spec

```diff
@@ -37,7 +37,15 @@ commands.
 %setup -q -n %{sname}-%{version}
 
 %build
-%configure PG_CONFIG=%{pginstdir}/bin/pg_config --with-extra-version="%{?conf_extra_version}"
+
+currentgccver="$(gcc -dumpversion)"
+requiredgccver="4.8.0"
+if [ "$(printf '%s\n' "$requiredgccver" "$currentgccver" | sort -V | tail -n1)" = "$requiredgccver" ]; then
+    echo ERROR: At least GCC version "$requiredgccver" is needed
+    exit 1
+fi
+
+%configure PG_CONFIG=%{pginstdir}/bin/pg_config --with-extra-version="%{?conf_extra_version}" CC=$(command -v gcc)
```

Comment:
I am curious, why do we need `CC=$(command -v gcc)` here?

Reply:
Mainly to be sure from the build output that we are using the right one. I don't think it's really needed per se, because `pg_config --cc` outputs `gcc --std=gnu99`. If it would use an absolute path though it would be an issue.